css/styles.css

```diff
@@ -216,9 +217,11 @@ button:active {
 #form-section {
   width: 100%;
   display: flex;
+  padding-bottom: 50px;
   justify-content: space-around;
-  background: url(../imgs/form_background.png) no-repeat;
-  background-position: right bottom;
+  background: url("../imgs/form-desktop.png");
+  background-size: cover;
+  background-position: bottom;
 }
 
 #form-section h2 {
```

Comment:
- [ ] Please only use one input for full name instead of two.
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/aec2f927-f149-4536-9752-79d3efacab11)